frontend/src/metabase/lib/i18n.js

```diff
@@ -119,7 +113,31 @@ function addMsgIds(translationsObject) {
   }
 }
 
-// set the initial localization
-if (window.MetabaseLocalization) {
-  setLocalization(window.MetabaseLocalization);
+// Runs `f` with the current language for ttag set to the instance (site) locale rather than the user locale, then
+// restores the user locale. This can be used for translating specific strings into the instance language; e.g. for
+// parameter values in dashboard text cards that should be translated the same for all users viewing the dashboard.
+export function withInstanceLocalization(f) {
```

Comment:
worth adding a test for this to check if the locale is reset after `f` is executed?

Reply:
Can't seem to do this from a unit test so I added it to the Cypress test.